api/proto/info/info.proto

```diff
@@ -394,6 +395,38 @@ message ZCellularProvider {
   bool roaming = 4;
 }
 
+// Snapshot states
+enum ZSnapshotState {
+  Z_SNAPSHOT_STATE_UNSPECIFIED = 0;
+  // This state is used when a snapshot is in the process of being
+  // created or an error occurred during the first attempt to create it.
+  // (For example, the operation was delayed)
+  Z_SNAPSHOT_STATE_CREATING = 1;
+  // This state is used when the snapshot has been successfully created.
+  Z_SNAPSHOT_STATE_CREATED = 2;
+  // This state is used when the snapshot is pending deletion or
+  // the first deletion attempt was not successful.
+  Z_SNAPSHOT_STATE_DELETING = 3;
+  // This state is used to send information to the controller about a
+  // snapshot that was implicitly deleted after a rollback snapshot
+  // or volume delete command. After sending a message with this status once,
+  // EVE no longer expects to receive the configuration (SnapshotConfig) for this snapshot.
```

Comment:
Can't make that assumption.
EVE-OS will ignore the snapshot config for the entries which have been implictly deleted.

Reply:
That's right, EVE-OS will ignore the snapshot configuration for entries that have been implicitly deleted. We can't do anything with a snapshot that doesn't exist.
But we can send a message about its status if for some reason the controller does not know that it was implicitly deleted.